### ISSUE TITLE
[kind] ERROR: unknown flag: --name

### ISSUE THREAD

**Issue by stv0g** [jupyterhub/zero-to-jupyterhub-k8s#1650]
### Bug description

Bootstrapping a development cluster with kind 0.7 fails with the following error:

```
`kind get kubeconfig-path --name jh-dev` errored (1)
ERROR: unknown flag: --name
```

### How to reproduce

1. Clone z2jh repo, branch master
2. Run `./dev kind create`

### Your personal set up

 - OS: MacOS, Docker Desktop
 
 - Version:
  - `kind v0.7.0 go1.13.6 darwin/amd64`
  - `docker 19.03.8`
 
 - Configuration: fresh checkout from master

**Comment by consideRatio** [comment]
Helm 3, but setup was designed for helm 2 =/

I'm not happy at all about the current state of using that script (`dev`), I'd say toss it and go minikube + helm3, or k3d + helm3.

**Comment by stv0g** [comment]
> Helm 3, but setup was designed for helm 2 =/

Is this really related to Helm? Seem's to me more like an issue if KinD?

I was trying to add IPv6 tests into Travis. But this seem to be hard than I imagined.. Travis does not support IPv6 :(

**Comment by consideRatio** [comment]
Ah woops okay, there may be an open an open pr about fixing this alreafy. Ibe not been able to keep up with review work etc and want to replace kind and the dev script so they have been given low prio

**Comment by bsamadi** [comment]
Any updates on this issue?

**Comment by consideRatio** [comment]
This issue will be closed by #1664

**Comment by consideRatio** [comment]
Closed by #1664!
